extension/chrome/elements/compose.htm

```diff
@@ -68,8 +68,22 @@ <h1>New Secure Message</h1>
     </tr>
     <tr id="password_or_pubkey_container">
       <td>
-        Emails <span class="grey">in grey</span> don't seem to use encryption. Add a <span class="label_password" style="display: block;">password
-          protection</span><span class="start">"</span><input id="input_password" placeholder="" tabindex="5" data-test="input-password"><span class="end">"</span>
+        Emails <span class="grey">in grey</span> don't seem to use encryption. Add a password to encrypt this message.
+        <div class="line">
+          <!-- Add a <span class="label_password" style="display: block;">password protection</span> -->
+          <div style=" display: inline-block;">
+            <input id="input_password" style="margin-bottom: 0px;" placeholder="" data-test="input-password">
+            <div class="password_bar">
+              <div style="height: 3px"></div>
+            </div>
+          </div>
+          <div style="display: inline-block;">
+            <div class="password_feedback"><span class="password_result"></span> (time to crack: <span class="password_time"></span>)
+              <ul></ul>
+            </div>
+          </div>
+        </div>
+
         <div class="question_copy">If they do use encryption, <a href="#" class="add_pubkey" data-test="action-open-add-pubkey-dialog">add
```

Comment:
please remove the css margin-top from question_copy (in the css file), it doesn't play well with the new design (gap too wide)